configure.ac

```diff
@@ -316,6 +318,7 @@ AS_IF([test "$enable_pmi_simple" = "yes"],
      [OPAL_CHECK_PMI(pmi_type)])
 AM_CONDITIONAL([USE_PMI_SIMPLE], [test "$enable_pmi_simple" = "yes"])
 AM_CONDITIONAL([USE_PMI2], [test "$pmi_type" = "pmi2"])
+AM_CONDITIONAL([USE_PMIX], [test "$opal_external_pmix_version_found" = 1])
```

Comment:
Oh wow, can multiple PMI options get enabled at the same time?

Reply:
Yes, I like to enable PMI, PMI2, *and* PMIX just because I enjoy the chaos.

But seriously, I suppose I should fix that.

Reply:
Let's file as an issue for future Dave.